pkg/kube/inject/inject.go

```diff
@@ -58,6 +59,8 @@ import (
 	yamlDecoder "k8s.io/apimachinery/pkg/util/yaml"
 )
 
+const annotationHoldApplicationUntilProxyStarts = "alpha.sidecar.istio.io/holdApplicationUntilProxyStarts"
```

Comment:
shouldnt it go to api repo?

Reply:
I wanted to keep the initial set of changes limited until I get the team's input on this approach. I'm now in the process of replacing the annotation with a field in ProxyConfig.

Reply:
Actually, I've kept the annotation for now, but I've moved this to the api repo.

Reply:
Annotation removed, as this is now configured via the `proxy.istio.io/config` annotation.